sssom/util.py

```diff
@@ -169,6 +176,8 @@ def load(self, filename) -> None:
         self.df = read_pandas(filename)
 
     def convert(self) -> Dict[str, Any]:
+        if self.df is None:
```

Comment:
why is the df even allowed to be none?

Reply:
I would prefer not to add to the complexity of this PR.. This is a separate issue